EIPS/eip-7557.md

```diff
@@ -0,0 +1,403 @@
+---
+eip: 7557
+title: Block-level Warming with fair cost savings
+description: Block-level warming of addresses and slots with access lists
```

Comment:
You have a bit more room (144 characters) in your description. You should expand and give a bit more detail.